docs/polkadot.rst

```diff
@@ -16,3 +16,4 @@ Polkadot
    polkadot/economics
    polkadot/security
    polkadot/Polkadot-Host-Spec
+   polkadot/Polkadot-Runtime-Spec
```

Comment:
The existing links are all top-level sections. So it might be better either to put this under a "specifications" directory, or just have 1 specifications page with multiple PDFs linked on it.

Reply:
Good point, I will turn this into a specification section instead (i.e. subfolder)

Reply:
You forgot to update this to just polkadot/specifications (and rm the previous line with Host-Spec) but otherwise it looks good!